rxnorm-integration/src/test/java/dev/ikm/tinkar/rxnorm/integration/RxnormDescriptionSemanticIT.java

```diff
@@ -0,0 +1,93 @@
+import dev.ikm.tinkar.entity.EntityService;
+import dev.ikm.tinkar.entity.EntityVersion;
+import dev.ikm.tinkar.entity.PatternEntityVersion;
+import dev.ikm.tinkar.entity.SemanticEntityVersion;
+import dev.ikm.tinkar.entity.SemanticRecord;
+import dev.ikm.tinkar.entity.SemanticVersionRecord;
+import dev.ikm.tinkar.terms.EntityFacade;
+import dev.ikm.tinkar.terms.EntityProxy;
+import dev.ikm.tinkar.terms.TinkarTerm;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.util.UUID;
+
+import static dev.ikm.tinkar.terms.TinkarTerm.DESCRIPTION_NOT_CASE_SENSITIVE;
+import static dev.ikm.tinkar.terms.TinkarTerm.FULLY_QUALIFIED_NAME_DESCRIPTION_TYPE;
+import static dev.ikm.tinkar.terms.TinkarTerm.REGULAR_NAME_DESCRIPTION_TYPE;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class RxnormDescriptionSemanticIT extends AbstractIntegrationTest {
+
+    /**
+     * Test RxnormDescriptions Semantics.
+     *
+     * @result Reads content from file and validates Description of Semantics by calling private method assertConcept().
+     */
+    @Test
+    public void testRxnormDescriptionSemantics() throws IOException {
+        String sourceFilePath = "../rxnorm-origin/";
+        String errorFile = "target/failsafe-reports/Rxnorm_Descriptions_not_found.txt";
+        String absolutePath = findFilePath(sourceFilePath, rxnormOwlFileName);
+        int notFound = processOwlFile(absolutePath, errorFile);
+
+        assertEquals(0, notFound, "Unable to find " + notFound + " Rxnorm Description semantics. Details written to " + errorFile);
+    }
+
+    @Override
+    protected boolean assertOwlElement(RxnormData rxnormData) {
+        if (rxnormData.getId() != null) {
+            // Generate UUID based on RxNorm ID
+            UUID conceptUuid = uuid(rxnormData.getId());
+
+//            EntityProxy.Concept concept = EntityProxy.Concept.make(PublicIds.of(conceptUuid));
+            EntityProxy.Semantic descSemantic =
+                    EntityProxy.Semantic.make(PublicIds.of(UuidT5Generator.get(UUID.fromString(namespaceString),
+                    (EntityProxy.Concept.make(PublicIds.of(conceptUuid))).publicId().asUuidArray()[0] + rxnormData.getRxnormName() + "DESC")));
+//            (concept.publicId().asUuidArray()[0] + rxnormData.getRxnormName() + "DESC")
+
+            StateSet state = StateSet.ACTIVE;
+            StampPositionRecord stampPosition = StampPositionRecord.make(timeForStamp, TinkarTerm.DEVELOPMENT_PATH.nid());
+            StampCalculator stampCalc = StampCoordinateRecord.make(state, stampPosition).stampCalculator();
+//            ConceptRecord entity = EntityService.get().getEntityFast(conceptUuid);
+//            SemanticRecord entity = EntityService.get().getEntity();
+//            SemanticRecord entity = EntityService.get().getEntityFast(conceptUuid);
+            SemanticRecord entity = EntityService.get().getEntityFast(descSemantic.uuids());
```

Comment:
Rather than looking up the Semantic directly, you can call forEachSemanticForComponentOfPattern method. See the LoincDescriptionPartSemanticIT in loinc-data.